test/extended/openstack/local-bd-etcd.go

```diff
@@ -0,0 +1,162 @@
+package openstack
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack/compute/v2/flavors"
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"github.com/stretchr/objx"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+)
+
+const minEtcdDiskSizeGiB = 7
+const etcdBlockDeviceName = "etcd"
+
+var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The OpenStack platform", func() {
+	defer g.GinkgoRecover()
```

Comment:
Please keep variables local to their context:

```suggestion
var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The OpenStack platform", func() {
	defer g.GinkgoRecover()

	const minEtcdDiskSizeGiB = 7
	const etcdBlockDeviceName = "etcd"
```

Reply:
No because they are used in functions too at the end of the file and if I move them, it doesn't compile.